### PR TITLE
🗃 💥: Renaming dt table

### PR DESCRIPTION
**Pull request by afitzek**
This brings the device type model into
it's final state.

Change-type: minor
Signed-off-by: Andreas Fitzek <andreas@balena.io>

### CODE REVIEW

File: src/migrations/00013-add-device-types.sql [hunk 1]
```diff
@@ -1,29 +1,28 @@
 -- create device types table
 
-CREATE TABLE IF NOT EXISTS "device type" (
+CREATE TABLE IF NOT EXISTS "device type table" (
```
Comment (dfunckt) [inline src/migrations/00013-add-device-types.sql]: Ouch. How did this slip though? Can you please make this a separate PR that is merged as soon as possible? The API is completely broken now.

Comment (afitzek) [reply src/migrations/00013-add-device-types.sql]: @dfunckt  
Here is the PR that fixes this: https://github.com/balena-io/open-balena-api/pull/179

I also realized that we have two 00014 migrations, one of them should become 00015.
